legend-depot-entities-services/src/main/java/org/finos/legend/depot/server/resources/entities/EntitiesDependenciesResource.java

```diff
@@ -73,6 +75,24 @@ public Response getEntitiesFromDependencies(@PathParam("groupId") String groupId
         return handle(GET_VERSION_DEPENDENCY_ENTITIES, () -> this.entitiesService.getDependenciesEntities(groupId, artifactId, versionId, transitive, includeOrigin), request, () -> EtagBuilder.create().withGAV(groupId, artifactId, versionId).build());
     }
 
+    @GET
+    @Path("/projects/{groupId}/{artifactId}/versions/{versionId}/classifier/{classifier}/dependencies")
+    @ApiOperation(GET_VERSION_DEPENDENCY_ENTITIES)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getEntitiesFromDependenciesByClassifier(@PathParam("groupId") String groupId,
+                                                            @PathParam("artifactId") String artifactId,
+                                                            @PathParam("versionId") @ApiParam(value = VersionValidator.VALID_VERSION_ID_TXT) String versionId,
+                                                            @PathParam("classifier") @ApiParam(value = VALID_CLASSIFIERS) String classifier,
+                                                            @QueryParam("transitive") @DefaultValue("false")
+                                                            @ApiParam("Whether to return transitive dependencies") boolean transitive,
+                                                            @QueryParam("includeOrigin") @DefaultValue("false")
+                                                            @ApiParam("Whether to return start of dependency tree") boolean includeOrigin,
+                                                            @Context Request request)
+    {
+        return handle(GET_VERSION_DEPENDENCY_ENTITIES, () -> this.entitiesService.getDependenciesEntitiesByClassifier(groupId, artifactId, versionId, classifier, transitive, includeOrigin), request, () -> EtagBuilder.create().withGAV(groupId, artifactId, versionId).withClassifier(classifier).build());
```

Comment:
etag might not completely work , if transitive/origin is true or false we should expect a diff response,
with this it might probably give me the same response irrespective transitive true/false.

Reply:
Seems that it should be fine by reading https://stackoverflow.com/questions/78770684/should-etags-be-unique-for-same-url-but-with-different-request-parameters    I've tested (only few times), it seems that I can get correct responses.   
May I know when/how you got a wrong response?